cluster/aws/templates/format-disks.sh

```diff
@@ -183,7 +193,9 @@ if [[ -n "${move_docker}" ]]; then
     mv /var/lib/docker ${move_docker}/
   fi
   mkdir -p ${move_docker}/docker
-  ln -s ${move_docker}/docker /var/lib/docker
+  if [[ ! -e /var/lib/docker ]]; then
```

Comment:
Is there a race here?  How can that directory ever exist, we just moved it away above on 193